lib/parsers/gopkg-parser.ts

```diff
@@ -0,0 +1,64 @@
+
+// https://golang.github.io/dep/docs/Gopkg.lock.html
+import {GoPackageConfig, LockedDeps} from '../types';
+import * as toml from 'toml';
+import {InvalidUserInputError} from '../errors';
+import {DepManifest, GopkgLockEntry} from './types';
+
+function parseDepLockContents(lockFileString: string): LockedDeps {
+    try {
+        const lockJson = toml.parse(lockFileString) as { projects: GopkgLockEntry[] };
+
+        const deps: LockedDeps = {};
+        if (lockJson.projects) {
+            lockJson.projects.forEach((proj) => {
+                const version = proj.version || ('#' + proj.revision);
+
+                proj.packages.forEach((subpackageName) => {
+                    const name =
+                        (subpackageName === '.' ?
+                            proj.name :
+                            proj.name + '/' + subpackageName);
```

Comment:
Here you could use template literal e.g. 
on line 15 ->  const version = proj.version || (`#${proj.revision}`);
on line 21 -> `${proj.name}/${subpackageName}`

Reply:
This file was merely moved, but w/o a logical change.
IMO once I'll enforce a prettier this will be fixed